src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form class="form search-box-big"
+    action="#" 
+    method="post"
+    data-qa="big"
+    >
+      <input
+      class="input input-big" 
```

Comment:
```suggestion
      class="input input-big" 
```
pay attention to the checklist